README.md

```diff
@@ -224,7 +232,7 @@ Maybe<string> apple = "apple";
 
 Maybe<string> favoriteFruit = apple.Where(IsMyFavorite);
 
-Console.WriteLine(favoriteFruit); // "No value"
+Console.WriteLine(favoriteFruit.ToString()); // "No value"
```

Comment:
I realized these would all throw if the `Maybe` is 'none' and we don't explicitly call `.ToString()`.

I'm used to always embedding my `Maybe` values in interpolated strings or using `Execute` to get the value vs printing the `Maybe` itself.